Assets/Scripts/WordHandler/WordStore.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+
+static class WordStore
```

Comment:
I'd stick it under /GlobalValues, just to keep globally accessible things in one place